config_template.yml

```diff
@@ -11,17 +11,22 @@ sra_ids:
 - ID3
 - ID4
 
-# Choose appropriate restriction sites
+# Restriction sites
 hicpro_restriction_sites: "T^TAA"
 
-# Choose appropriate resolution
+# Ligation site
+hicpro_ligation_site: "AAGCTAGCTT"
```

Comment:
Should the `hicpro_ligation_site` parameter be mandatory?

Reply:
No, it can be optional. The documentation in HiC Pro says "Note that if if the LIGATION_SITE parameter in the not defined, HiC-Pro will skip the second step of mapping" ([https://nservant.github.io/HiC-Pro/MANUAL.html](https://nservant.github.io/HiC-Pro/MANUAL.html)).

Reply:
OK. I made it more clear in the YAML parameter file and in the `Snakefile`